application/src/main/java/org/thingsboard/server/service/state/DefaultDeviceStateService.java

```diff
@@ -253,16 +253,17 @@ void updateActivityState(DeviceId deviceId, DeviceStateData stateData, long last
                 onDeviceActivityStatusChange(deviceId, true, stateData);
             }
         } else {
-            log.debug("updateActivityState - fetched state IN NULL for device {}, lastReportedActivity {}", deviceId, lastReportedActivity);
+            log.debug("updateActivityState - fetched state IS NULL for device {}, lastReportedActivity {}", deviceId, lastReportedActivity);
```

Comment:
I do not have it available here, because the only place where tenant id is available is state data and this else branch covers the case when mentioned state data is null